src/tests/multihost/alltests/test_automount.py

```diff
@@ -194,6 +194,113 @@ def test_004_autofsnone(self, multihost, set_autofs_search_base,
         cmd = multihost.client[0].run_command(nfs_test, raiseonerr=False)
         assert cmd.returncode == 0
 
+    @pytest.mark.tier1_2
+    def test_two_automount_maps(self, multihost,
+                                backupsssdconf):
+        """
+        :title: Automount sssd issue when 2
+         automount maps have the same key
+         (one un uppercase, one in lowercase)
```

Comment:
fix typo, or change title to
"Automount sssd issue when 2 maps have same key in different case"